sqe-dtos/Artefact.cs

```diff
@@ -26,18 +27,20 @@ public class ArtefactListDTO
 		public List<ArtefactDTO> artefacts { get; set; }
 	}
 
+	// TODO: Should we make the mask and positioning data a PolygonDTO here, like it is in the GET DTO?
 	public class UpdateArtefactDTO
 	{
 		public string mask { get; set; }
 		public string name { get; set; }
-		public string position { get; set; }
+		public float? scale { get; set; }
+		public float? rotate { get; set; }
+		public uint? translateX { get; set; }
```

Comment:
These *definitely* need an OffsetDTO (or TranslationDTO) class. Have two properties in the model is an internal implementation detail, but this is the external API. Grouping the two properties makes sense - they always come together. Plus, as I've written in the comment for the model - they must be both null or both not null, you can't have X null and Y not null. Also, are we sure they can't be negative?

Reply:
I will update accordingly.